systemd/pfl.service

```diff
@@ -4,6 +4,7 @@ After=network.target
 
 [Service]
 Type=oneshot
+User=portage
```

Comment:
How about adding `Group=portage`? That would be in line with [`cron/pfl`](https://github.com/portagefilelist/client/blob/master/cron/pfl).

----

PS: @tuxmainy `os.path.expanduser(..)` would work even in absence of `${HOME}`:

```console
# ( unset HOME ; python -c 'import os; print(os.getenv("HOME"), os.path.expanduser("~/foo/bar"))' )
None /home/user123/foo/bar
```

I can make a pull request about just that, if it's interesting.

Reply:
Makes sense. Will update pull request. Would make sense to update the python code too. Not sure if it will be merged though. This pull request has been open for quit ea while.